monai/bundle/workflows.py

```diff
@@ -225,6 +225,115 @@ def check_properties(self) -> list[str] | None:
         return [n for n, p in self.properties.items() if p.get(BundleProperty.REQUIRED, False) and not hasattr(self, n)]
 
 
+class PythonicWorkflow(BundleWorkflow):
+    """
+    Base class for the pythonic workflow specification in bundle, it can be a training, evaluation or inference workflow.
+    It defines the basic interfaces for the bundle workflow behavior: `initialize`, `run`, `finalize`, etc.
+    And also provides the interface to get / set public properties to interact with a bundle workflow.
+
+    Args:
+        workflow_type: specifies the workflow type: "train" or "training" for a training workflow,
```

Comment:
I think for pythonic workflow, we can also enable read config file contents.
It's common to prepare a config file to save hyper parameters, otherwise it's not convenient to adjust them.

Reply:
we can discuss offline about it

Reply:
added in the latest change, please help review it again, thanks.